apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -268,6 +271,9 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       },
     });
 
+    // Submit Transcription Batch Processor Job
+    await submitBatchProcessorTranscriptionJob(recording_id);
```

Comment:
This api endpoint is executed when recording has been stopped and ready to download.

1) submit transcription job 
2) fetch transcription download link from API endpoint (`v1/bookings/[bookingId]/transcripts/[recordingId]`)

Reply:
For clarification, is the transcription going to be available through the `download link` of the recording, or is it going to be its own separate link that we or the user has to obtain?

Reply:
In response to this comment: https://github.com/calcom/cal.com/pull/15358#issuecomment-2154401554

If this throws an error, can we ensure we're catching them properly so it doesn't stop the sending of emails in the next line?